src/debug/i2cScan.h

```diff
@@ -104,6 +105,15 @@ void scanI2Cdevice(void)
                 DEBUG_MSG("axp192 PMU found\n");
             }
 #endif
+        if (addr == BME_ADDR || addr == BME_ADDR_ALTERNATE) {
+            // Assume BME680 first until we find out otherwise in environmental telemetry init
```

Comment:
Is there some sub register you can use here to tell 280 and 680 apart? Like when we scan for cardkb or rak-kb. If not, we can leave it like it is with the detect later.

Reply:
My google-fu didn't come up with anything. I'll skim the Bosch docs really quick to see if I can dig up something there